recipe/meta.yaml

```diff
@@ -387,6 +387,7 @@ outputs:
         - cmake
       host:
         - {{ pin_subpackage('libopenvino', exact=True) }}
+        - gflags
```

Comment:
why is it required? only for tools?
I vote for excluding tools from conda build then

Reply:
no, after excluding the tools from the build